### PR TITLE
Added npi_gpio component section

### PR DESCRIPTION
**Pull request by Tungsteno74**
## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/npi_gpio.markdown [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+comments: false
+sharing: true
+footer: true
+logo: nanopi-logo.png
+ha_category:
+  - DIY
+  - Binary Sensor
+  - Cover
+  - Switch
+ha_release: pre 0.93.2
```
Comment (klaasnicolaas) [inline source/_components/npi_gpio.markdown]:
```suggestion
ha_release: 0.94
```

File: source/_components/npi_gpio.markdown [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+ha_iot_class: Local Push
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/npi_gpio.markdown]: This is only for existing pages

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by Tungsteno74** [comment]
@klaasnicolaas thanks for your review

**Comment by klaasnicolaas** [comment]
Well @Tungsteno74 it's my duty 😜

File: source/_components/npi_gpio.markdown [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+comments: false
+sharing: true
+footer: true
+logo: nanopi-logo.png
+ha_category:
+  - DIY
+  - Binary Sensor
+  - Cover
+  - Switch
+ha_release: 0.94
```
Comment (klaasnicolaas) [inline source/_components/npi_gpio.markdown:16]:
```suggestion
ha_release: 0.96
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed.

**Pull request closed without merging**